package.json

```diff
@@ -23,39 +23,48 @@
     "@types/react": "16.9.17",
     "@types/react-color": "3.0.1",
     "@types/react-dom": "16.9.4",
+    "eslint-plugin-json-format": "2.0.1",
     "husky": "4.0.10",
     "lint-staged": "10.0.0",
     "node-sass": "4.13.1",
     "prettier": "1.19.1",
-    "rewire": "^4.0.1",
+    "rewire": "4.0.1",
     "typescript": "3.7.5"
   },
   "eslintConfig": {
-    "extends": "react-app"
+    "extends": "react-app",
+    "plugins": [
+      "json-format"
+    ],
+    "settings": {
+      "json/sort-package-json": false
```

Comment:
if we can sort it automatically.. why not?

Reply:
It sorts every key, not just deps. Also not sure how it behavior when linting just staged (via `lint-staged`).

Reply:
What's worse, it doesn't sort it. It just errors out it's not sorted. Let's skip this for now.